examples/persistent-counters/gserver.go

```diff
@@ -47,7 +47,7 @@ func (c *CounterServer) Get(ctx context.Context, rpcReq *pb.CounterGetReq) (*pb.
 func (c *CounterServer) Inc(ctx context.Context, rpcReq *pb.CounterIncReq) (*pb.CounterIncRes, error) {
 	ch := make(chan interface{}, c.resBufSize)
 	req := &Request{
-		K: string(charset[rand.Int() % len(charset)]),
+		K: string(charset[time.Now().UnixNano() % 26]),
```

Comment:
You have very small number of keys which is creating contention. Atleeast rotate between 1m keys